### PR TITLE
Bug fixes on Checklist page

### PR DESCRIPTION
**Pull request by CurtisUnderwood**
### Description

List of proposed changes:

- Checklist page bug where if the Important Terms and Filter Tasks section were opened first they would cause the accordions to open as well has been fixed. Can be re-produced by opening one of these sections while accordions are closed.
image

- Added transition effect on the Accordion opening (not sure if this was removed intentionally?)
- Removed focus outline on accordion for consistency with other Collapse elements

### Additional Notes

### CODE REVIEW

**Review by sebastien-comeau — CHANGES_REQUESTED** [review]

File: frontend/src/components/TaskGroupAccordion.tsx [hunk 1]
```diff
@@ -59,25 +59,27 @@ export const TaskGroupAccordion: React.FC<TaskGroupAccordionProps> = ({
...
-              key={task.id}
-              expanded={expandedTasks.includes(task.id)}
-              onTaskAccordionChange={onTaskAccordionChange}
-              linksHeader={linksHeader}
-              srTag={srTag}
-              {...task}
-            />
-          ))}
-        </section>
+        <Collapse in={expanded} mountOnEnter unmountOnExit>
```
[the remark this answers is not in the document]
Comment (Fbasham) [reply frontend/src/components/TaskGroupAccordion.tsx:69]: unless we right our own CSS animation ;)

File: frontend/src/components/TaskGroupAccordion.tsx [hunk 2]
```diff
@@ -59,25 +59,27 @@ export const TaskGroupAccordion: React.FC<TaskGroupAccordionProps> = ({
         open={disabled ? false : expanded}
         data-gc-analytics-expand={sectionTitle}
       >
-        <summary className="flex cursor-pointer  gap-2 bg-primary-800 px-4 py-5 text-white focus:outline focus:outline-2 focus:outline-offset-2 focus:outline-primary-700 group-aria-disabled/task-group:pointer-events-none group-aria-disabled/task-group:select-none">
+        <summary className="focus:outline-0 flex cursor-pointer  gap-2 bg-primary-800 px-4 py-5 text-white focus:outline focus:outline-2 focus:outline-offset-2 focus:outline-primary-700 group-aria-disabled/task-group:pointer-events-none group-aria-disabled/task-group:select-none">
```
[the remark this answers is not in the document]
Comment (CurtisUnderwood) [reply frontend/src/components/TaskGroupAccordion.tsx:62]: image
Keeping the focus outline looks like this, not needed in my opinion unless there's a a11y reason.

File: frontend/src/pages/checklist/[filters].tsx [hunk 3]
```diff
@@ -262,49 +262,51 @@ const ChecklistResults: FC<ChecklistResultsProps> = ({
...
-              tasks={receivingBenefits.tasks.filter((task) => filterTasksByTag(task, filters))}
-            />
-            <div className="mt-4 lg:hidden">
-              <Button variant="text" startIcon={<Cached />} size="large" onClick={handleOnRestartQuizClick}>
-                {t('restart-quiz')}
-              </Button>
-            </div>
-          </section>
+          <div className='lg:col-span-8 xl:col-span-9'>
+            <section id="content" className="print-href grid gap-4">
```
[the remark this answers is not in the document]
Comment (CurtisUnderwood) [reply frontend/src/pages/checklist/[filters].tsx]: Fixed, good catch

File: frontend/src/components/TaskGroupAccordion.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (CurtisUnderwood) [reply frontend/src/components/TaskGroupAccordion.tsx]: Fixed

File: frontend/src/components/TaskGroupAccordion.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (sebastien-comeau) [reply frontend/src/components/TaskGroupAccordion.tsx:62]: If you need to remove the focus style, remove all of them in this case.

[the remark this answers is not in the document]
Comment (Fbasham) [reply frontend/src/components/TaskGroupAccordion.tsx:62]: there is an a11y implication actually https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html

[the remark this answers is not in the document]
Comment (CurtisUnderwood) [reply frontend/src/components/TaskGroupAccordion.tsx]: Fixed this

**Pull request closed without merging**